cmd/lk/room.go

```diff
@@ -177,6 +177,11 @@ var (
 							Usage:     "read attributes from a `JSON` file",
 							TakesFile: true,
 						},
+						&cli.BoolFlag{
+							Name:  "auto-subscribe",
+							Usage: "Automatically subscribe to published tracks. Default is false.",
+							Value: false,
```

Comment:
i'm hesitant to change the default behavior in this PR. maybe best as true

Reply:
Seconded. Also, when `Value` is set on a flag, the CLI will display `(defaault: $value)` so you can remove it from the usage string.